packages/core/src/components/button/_common.scss

```diff
@@ -17,16 +17,21 @@ $button-icon-spacing-large: ($pt-button-height-large - $pt-icon-size-large) / 2
 $button-color-disabled: $pt-text-color-disabled !default;
 $dark-button-color-disabled: $pt-dark-text-color-disabled !default;
 
-$button-border-color: rgba($black, 0.1) rgba($black, 0.1) rgba($black, 0.17) rgba($black, 0.1) !default;
-$button-border-color-hover: rgba($black, 0.2) rgba($black, 0.2) rgba($black, 0.27) rgba($black, 0.2) !default;
-$button-border-color-active: rgba($black, 0.35) rgba($black, 0.25) rgba($black, 0.25) rgba($black, 0.25) !default;
+$button-border-color:
+  rgba($black, 0.1) rgba($black, 0.1) rgba($black, 0.17) rgba($black, 0.1) !default;
+$button-border-color-hover:
+  rgba($black, 0.2) rgba($black, 0.2) rgba($black, 0.27) rgba($black, 0.2) !default;
+$button-border-color-active:
+  rgba($black, 0.35) rgba($black, 0.25) rgba($black, 0.25) rgba($black, 0.25) !default;
```

Comment:
Opining after the fact. In previous projects, I've line-wrapped in a way that aligns parallel structures, because it makes differing values easier to detect. For instance:

```
$button-border-color-active:
  rgba($black, 0.35)
  rgba($black, 0.25)
  rgba($black, 0.25)
  rgba($black, 0.25)
  !default
;
```

Of course, there's a tradeoff in line-count here. That tends to garner derision, but I like the cleanliness of the approach. Just a thought.